modules/caddyhttp/encode/br/brotli_precompressed.go

```diff
@@ -0,0 +1,31 @@
+package caddybr
```

Comment:
Thinking this should be in a folder called `brotli` (it's not a long name), and maybe `caddybrotli`, but that's just my opinion.

Reply:
thought I changed that. It should be `caddybrotli`.